sandboxmgt/app/templates/my_sandbox.html

```diff
@@ -7,12 +7,14 @@
 
   <div class="container">
     <section class="content-section">
-      {% blocktrans %}
-        <h2 class="content-section__title">My sandbox</h2>
-        <p class="content-section__body">
-         Placeholder
-        </p>
-      {% endblocktrans %}
+      <h2 class="content-section__title">My sandbox</h2>
+      <p class="content-section__body">
+        {% if user and user.is_authenticated %}
+          <a href="https://{{ user.username }}-rstudio.tools.sandbox.data-science.org.uk">
+            Open My Sandbox
```

Comment:
Capitals again. I think it's a bit old-school to have verbs (e.g. "click here") in link text - they are usually nouns. How about: "Your sandbox contains: <a>R Studio</a>".